internal/transport/transport_test.go

```diff
@@ -90,6 +90,7 @@ const (
 	invalidHeaderField
 	delayRead
 	pingpong
+	goAwayFrameSize = 42
```

Comment:
suggestion: these const values seems to be related for an enum. May be better to have separate const for goAwayFrameSize